addons/im_livechat/static/src/embed/common/persona_model_patch.js

```diff
@@ -0,0 +1,11 @@
+import { patch } from "@web/core/utils/patch";
+import { Persona } from "@mail/core/common/persona_model";
+
+patch(Persona.prototype, {
+    getNameForChannel(channel) {
+        if (channel.channel_type !== "livechat" || !this.user_livechat_username) {
+            return super.getNameForChannel(channel);
+        }
+        return this.user_livechat_username;
+    },
```

Comment:
Note: we already have `name` getter on channel member too, it would be nice to not duplicate too much logic.

Reply:
And also `authorName` on message...

Reply:
`displayName` override on thread